client/stylesheets/_responsive.scss

```diff
@@ -244,15 +350,18 @@
     // .homeCardLi {
     // 	width: 40%;
     // }
+    .searchResultsCardImg {
+        height: 145px !important;
+    }
 }
 
+
 @media (max-width: 580px) {
-    // .profileCardLi {
-    // 	width: 60% !important;
-    // }
+    .searchResultsCardImg {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `searchResultsCardImg` should be written in lowercase with hyphens